package.json

```diff
@@ -46,9 +46,12 @@
     "prop-types": "^15.6.2",
     "react": "^16.7.0",
     "rimraf": "^2.6.2",
-    "rollup": "^0.52.3",
+    "rollup": "^1.3.1",
     "rollup-plugin-babel": "^3.0.4",
-    "rollup-plugin-uglify": "^2.0.1"
+    "rollup-plugin-commonjs": "^9.2.1",
+    "rollup-plugin-node-resolve": "^4.0.1",
+    "rollup-plugin-replace": "^2.1.0",
+    "rollup-plugin-uglify": "^6.0.2"
```

Comment:
Terser plugin is more stable

Reply:
rollup-plugin-uglify is using uglify@3 which should be rather stable, right? it also acts here as additional smoke test against shipping es6+ code to npm (as uglify only understands es5 it would fail when seeing modern syntax)

Reply:
Plugin is more stable :)
Afaik terser have a option for ecmascript version